Index.html

```diff
@@ -0,0 +1,200 @@
+                 As the main venue for new media art production in
+                 Korea, Nabi promotes cross-disciplinary collaboration
+                 and understanding among science technology,
+                 humanities, and the arts.
+              </p>
+            </div>
+            </div>
+              <div class="HeaderTitle">
+                <div>
+              <img class="image" src="Asset/feat img/lila.JPG" alt="pason" />
+            </div>
+            <div>
+              <h3 class="TonicEnd">Lila Tretikov</h3>
+            <p6 class="Content">
+             Exective Director of the Wikimedia Foundation
+            </p6>
+            <p class="pop">
+                Lila Tretikov is the Exactive Director of the Wikimedia
+                Foundation, the nonorofit organization that operates
+                Wikimedia. Wikimediais freely available in 290 languages and used
+                by nearly half a billion people around the 
+                world every month.
+            </p>
+          </div>
+              </div>
+          </div>
+            </div>
+            <div class="second">
+            <div class="HeaderTitle">
+              <div>
+            <img class="image" src="Asset/feat img/kilnam chon.JPG" alt="pason" />
+          </div>
+          <div>
+            <h3 class="TonicEnd">Kilnam Chon</h3>
+          <p6 class="Content">
+            _________
+          </p6>
+          <p class="pop">
+              Kilnam Chon helped bring the internet to Asia and is an
+              outspoken advocate for the open web and digital commons in 2012, he was inducted
+               into the inaugural class  of the internet Society's (ISOC)internet Hall of fame.
+          </p>
+          </div>
+        </div>
+        <div class="HeaderTitle">
+          <div>
+          <img class="image" src="Asset/feat img/julia leda.jpg" alt="pason" />
+        </div>
+        <div>
+          <h3 class="TonicEnd">Julia Leda</h3>
+        <p6 class="Content">
+         President of Young Pirates of Europe
+        </p6>
+        <p class="pop">
+            European ingetration, political democracy and participation
+            of the youth through online as her major condem,<br>
+            Reda's report outlining potential changes to EU copyright<br>
+            law was approved by the Parliament in july.
+        </p>
+      </div>
+        </div>
+        <div class="HeaderTitle">
+          <div>
+        <img class="image" src="Asset/feat img/ryanmerkley.jpg" alt="pason" />\
+      </div>
+      <div>
+        <h3 class="TonicEnd">Ryan Merkley</h3>
+      <p6 class="Content">
+        CEO of creative commons, ex COO of the <br>
+        Mozzilla Foundation
+      </p6>
+      <p class="pop">
+          Ryan had been leading open-source projects at the <br>
+          Mozzilla Foundation such as the open source movement
+      </p>
+    </di>
+    </div>
+      </div>
+    </div>
+       </section>
```

Comment:
As specified in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/d047e7a9f19b5c63da497d1523293ddce5fd22f5/capstone/html_capstone.md?plain=1#L69), kindly ensure that the featured speakers are rendered from JavaScript.